tsconfig.json

```
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "esnext"
+    ],
+    "module": "NodeNext",
+    "allowJs": false,
+    "strict": false,
```

We should make this `strict` for better TS support.

```suggestion
    "strict": true,
```

---

I believe this was causing some issues when I had it enabled but I'll try re-enabling it now that this is in a better spot